lib/groups.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/**
+ * API Wrapper for the Gambit Groups API.
+ */
+
+const superagent = require('superagent');
+const logger = app.locals.logger;
+
+const uri = process.env.GAMBIT_GROUPS_API_URI;
+const apiKey = process.env.GAMBIT_GROUPS_API_KEY;
+const apiHeaderName = 'x-gambit-group-api-key';
+
+/**
+ * Make a GET request to Gambit Groups API.
+ * Handles authentication & URL formation.
+ * @param  {string} path API Path. eg: /group
+ * @return {Promise}
+ */
+function get(path) {
```

Comment:
You may want to call it from other module in this lib. Anyway, I see no reason to explicitly make them private.